sw/device/silicon_creator/lib/crc32_unittest.cc

```diff
@@ -27,21 +28,59 @@ INSTANTIATE_TEST_SUITE_P(AllCases, CrcTest,
                              TestParams{
                                  "The quick brown fox jumps over the lazy dog",
                                  0x414fa339,
+                             },
```

Comment:
nit: Leave a comment describing how the test values were generated?

e.g.
```
\```python
import zlib
hex(zlib.crc32(b"<string>"))
\```
```

Reply:
SG, will do.